### DISCUSSION TITLE
IDE start page to be edited by users?

### DISCUSSION THREAD

**Discussion by HFolkertH** [cincheo/daquota#147] [Q&A]
On the IDE start page I can use the pencil button to start editing the Model of the IDE start page itself. Is that on purpose?  Can i create my own version of the IDE start page?

![Image 2023-03-08 08-20-40](https://user-images.githubusercontent.com/22573455/223648163-cbc83eed-8abb-45ef-9bf3-0585e2436219.png)

I tried that pencil button because I want to add a picture to one of my applications. But it seems that I can only add a picture when i create  a NEW project/application. Afterwards is not possible?

By the way: i like the new default theme. It gives Dlite a modern feeling.

Folkert

**Comment by renaudpawlak** [comment]
I have kept the project manager editable for my own use because it is still under development and it is convenient to try things out easily. I will eventually turn it of.... All the changes you could make will not be saved anyway :) One day, there will be a customizable desktop app for dLite, but that day has not come yet.

Note that it's starting to be stable again... It was a tough refactoring and sorry if you have experienced weird effects.

Yes, I like the new theme too :) And the landing page has the same one too: https://dlite.io

It is starting to look better ;)
